src/emulator/gui/src/game_selector.cpp

```diff
@@ -142,6 +141,13 @@ void draw_game_selector(HostState &host) {
                 break;
             }
         }
+        ImGui::PushStyleColor(ImGuiCol_Text, GUI_COLOR_SEARCH_BAR_TEXT);
+        ImGui::PushStyleColor(ImGuiCol_FrameBg, GUI_COLOR_SEARCH_BAR_BG);
+        ImGui::SameLine(display_size.x - 515);
```

Comment:
I don't really accept this, sorry, i know this also subtract padding but it also contains hardcoded text width. It may breaks with other font in the future.

Use ImGui::CalcTextSize to get text size, than get the position by:
```cpp
display_size.x - (ImGui::CalcTextSize("Search").x + padding + text_box_width)
```